test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/Rendering/DefaultTemplatesUtilities.cs

```diff
@@ -335,6 +329,12 @@ private static ICompositeViewEngine CreateViewEngine()
             return viewEngine.Object;
         }
 
+        public static string FormatOutput(IHtmlHelper helper, object model)
+        {
+            var modelExplorer = helper.MetadataProvider.GetModelExplorerForType(model.GetType(), model);
+            return FormatOutput(modelExplorer);
+        }
+
```

Comment:
I could revert changes to this file but like leaving this method next to the other `FormatOutput()` overload.